src/screens/CreatePost/index.js

```diff
@@ -882,8 +861,8 @@ const styles = StyleSheet.create({
     justifyContent: 'flex-start',
     overflow: 'scroll'
   },
-  hastagText: {
-    color: COLORS.gray110,
+  hashtagText: {
+    color: COLORS.gray410,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes here seem to be related to updating the style of the `hashtagText`. Make sure that these changes align with the overall design and style guide of your application.
```diff
- hastagText: {
-    color: COLORS.gray110,
+ hashtagText: {
+    color: COLORS.gray410,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->